src/api/web-extension.js

```diff
@@ -58,10 +91,18 @@ export default {
     return true;
   },
 
-  // updates the overall Ion enrollment in the add-on, if needed.
-  // NOTE: if updating in the app store in the add-on suffices,
-  // this should probably just return the OK signal.
-  async updateIonEnrollment(studyID, enroll) {
-    return true;
+  /**
+   * Updates the overall Ion enrollment in the add-on.
+   * NOTE: if updating in the app store in the add-on suffices,
+   * this should probably just return the OK signal.
+   *
+   * @param {Boolean} enroll
+   *        `true` if user decided to enroll in the Ion platform,
+   *        `false` otherwise.
+   * TODO: Why can `enroll` be undefined?
```

Comment:
@hamilton hey! In `create-store.js` (`updateIonEnrollment`) we default `enroll = undefined`. When can that be undefined?

Reply:
I have no feeling about this. Please feel free to implement whatever you want in `web-extension.js` and have the return value be whatever format you prefer. We can then check that in `create-store.js` to make sure the frontend can respond appropriately. I have no guidance here other than feel free to own it.

Reply:
Oh, I see. So it toggles if nothing is provided, otherwise it reflects what's in `enroll`! Thanks for clarifying, that's much clearer, I'll add some documentation so that I don't forget :)

Reply:
Works for me!